### ISSUE TITLE
GaxError

### ISSUE THREAD

**Issue by jairobluecaribu** [googleads/google-ads-ruby#41]
Hello, 
Sometimes I get this error:
GaxError Exception occurred in retry method that was not classified as transient, caused by undefined method `each' for #<String:0x0000000004146118>
when I try  to get reports

it seems to be random.

**Comment by mcloonan** [comment]
I've never seen this error. Can you provide a sample request that you're making where you have seen this?

**Comment by xto** [comment]
Hello,

We're getting the same error whenever we make a `GoogleAdsService#search` call and we hit a `GRPC::DeadlineExceeded` exception.

It looks like there might be an issue in how GRPC errors are handled. Here's the stacktrace we're getting:

```
Google::Gax::RetryError GaxError Exception occurred in retry method that was not classified as transient, caused by undefined method `each' for #<String:0x00007f701c143b18> 
    vendor/bundle/ruby/2.5.0/gems/google-gax-1.5.0/lib/google/gax/api_callable.rb:369 rescue in block in retryable
    vendor/bundle/ruby/2.5.0/gems/google-gax-1.5.0/lib/google/gax/api_callable.rb:358 block in retryable
    vendor/bundle/ruby/2.5.0/gems/google-gax-1.5.0/lib/google/gax/api_callable.rb:147 start
    vendor/bundle/ruby/2.5.0/gems/google-gax-1.5.0/lib/google/gax/api_callable.rb:260 call
    vendor/bundle/ruby/2.5.0/gems/google-gax-1.5.0/lib/google/gax/api_callable.rb:260 block in create_api_call
    vendor/bundle/ruby/2.5.0/gems/google-ads-googleads-1.0.0/lib/google/ads/google_ads/v1/services/google_ads_service_client.rb:249 search
```

**Comment by evgeniy-b** [comment]
Hello,
Looks like I'm experiencing the same problem and from I found in the code it seems that happens [here](https://github.com/googleads/google-ads-ruby/blob/master/lib/google/ads/google_ads/logging_interceptor.rb#L74) because of assumption that `status_details` is an array when present. But it also may be a string: [assignment](https://github.com/googleapis/gax-ruby/blob/master/lib/google/gax/errors.rb#L46) <- [string value](https://github.com/googleapis/gax-ruby/blob/master/lib/google/gax/grpc.rb#L59)

**Comment by samphippen** [comment]
Fixed by e276cc6f29b98726234591513372d708207b752d

**Comment by evgeniy-b** [comment]
Almost :)
There is a type in https://github.com/googleads/google-ads-ruby/commit/e276cc6f29b98726234591513372d708207b752d#diff-37ebfa77114b5d33ba731f9afc95f72eR69 - triple _r_ in _error_
